module/documents/actor/actor.mjs

```diff
@@ -1007,7 +1008,7 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
       isBar: true
     }, updates) === false ) return this;
```

Comment:
Relatedly, I'm not sure why we have this hook call. Far as I can tell, it does nothing at all.

Reply:
As far as I understand it, that is so that anyone monitoring for changes to attributes on the token bar will see changes to HP made elsewhere, but it has been around forever so I don't really know `¯\_(ツ)_/¯`

Reply:
Let's leave it since we're all emotionally attached to it.